hub/test/src/testHttp.js

```diff
@@ -139,3 +227,9 @@ export function testHttp() {
       })
   })
 }
+
+export function testHttp() {
+  testHttpWithAzure()
+  testHttpDriverOption()
+  testHttpWithInMemoryDriver()
```

Comment:
If we decide to drop the in-memory driver, then we should consider using a test with the disk driver (in a random direcory) instead.

Reply:
Agreed. These tests are for http not any specific driver, the existing tests just happened to be using the mock-azure config, so I named the function appropriately.